examples/index.tsx

```diff
@@ -311,7 +311,19 @@ const app = (
           <br></br>
           <Box sx={{ width: "300px" }}>
             <Card title={"Example Card"}>
-              <Typography>This is a card</Typography>
+              <Typography>This is a basic card.</Typography>
+            </Card>
+          </Box>
+          <br></br>
+          <Box sx={{ width: "300px" }}>
+            <Card title={"Example Card"} closeButton={true}>
```

Comment:
fwiw  think the linter will normally want just `isPinned` or `closeButton` (without the ={true}) for attributes where they're true